tests/controller-test.js

```diff
@@ -244,6 +245,7 @@ OO = {
     Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', elementId, {skin:{config:{}}}, persistentSettings);
     Html5Skin.onSkinMetaDataFetched.call(controllerMock, 'customerUi', {});
     Html5Skin.onAttributesFetched.call(controllerMock, 'customerUi', {"attributes":{"provider":{"ots_stretch_to_output":"true"}}});
+    Html5Skin.trySetAnamorphicFixState.call(controllerMock, true);
```

Comment:
I think our unit tests for the skin controller need some improvement. I spent some time trying to write some tests for the scenarios in this ticket, but it looks like we don't currently have the setup required to test the actual logic of the controller. 

I think that creating a new test environment for the controller will need to be handled as a separate task.